pkg/whiteblacklist/whiteblacklist_test.go

```diff
@@ -118,3 +119,46 @@ func TestExclude(t *testing.T) {
 		}
 	})
 }
+
+func TestStatus(t *testing.T) {
+	t.Run("status when whitelist has single item", func(t *testing.T) {
+		item1 := "item1"
+		whitelist, _ := New(map[string]struct{}{item1: {}}, map[string]struct{}{})
+		actualStatusString := whitelist.Status()
+		expectedStatusString := "whitelisting the following items: " + item1
+		if actualStatusString != expectedStatusString {
+			t.Errorf("expected status %q but got %q", expectedStatusString, actualStatusString)
+		}
+	})
+	t.Run("status when whitelist has multiple items", func(t *testing.T) {
+		item1 := "item1"
+		item2 := "item2"
+		whitelist, _ := New(map[string]struct{}{item1: {}, item2: {}}, map[string]struct{}{})
+		actualStatusString := whitelist.Status()
+		expectedRegexPattern := `^whitelisting the following items: item1|item2, item2|item1$`
```

Comment:
`^whitelisting the following items: item1|item2, item2|item1$` -> `^whitelisting the following items: (item1|item2), (item2|item1)$`. Otherwise `^whitelisting the following items: item1|item2, item2|item1$` will not match `whitelisting the following items: item1`